lra/coordinator/server/src/main/java/io/helidon/lra/coordinator/CoordinatorService.java

```diff
@@ -149,6 +149,39 @@ public void routing(HttpRules rules) {
                 .put("/{LraId}/remove", this::leave);
     }
 
+    /**
+     * Get LRA by lraId.
+     *
+     * @param lraId without coordinator uri prefix
+     * @return LRA when managed by this coordinator or null
+     */
+    public Lra getLra(String lraId) {
+        return this.lraPersistentRegistry.get(lraId);
+    }
+
+    LazyValue<URI> getCoordinatorURL() {
```

Comment:
This method should be called `coordinatorUrl` to be aligned with Helidon conventions